Gems/ScriptCanvasPhysics/Code/Source/World.h

```diff
@@ -95,3 +96,5 @@ namespace ScriptCanvasPhysics
             AZ::EntityId ignore);
     } // namespace WorldFunctions
 } // namespace ScriptCanvasPhysics
+
+#include <Source/World.generated.h>
```

Comment:
The finall newline was removed from this file.

It should be added back.
```suggestion
#include <Source/World.generated.h>

```